src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/model/courtcaseservice/CCSDefendant.java

```diff
@@ -62,7 +63,7 @@ public static CCSDefendant of(Defendant defendant) {
                 .cro(defendant.getCro())
                 .preSentenceActivity(defendant.getPreSentenceActivity())
                 .suspendedSentenceOrder(defendant.getSuspendedSentenceOrder())
-                .sex(defendant.getSex())
+                .sex(Sex.getNormalisedSex(defendant.getSex()).toString())
```

Comment:
There's an alternative way that we could do this, i.e. on the way in when we map from the incoming payloads to the domain model [here](https://github.com/ministryofjustice/court-case-matcher/blob/main/src/main/java/uk/gov/justice/probation/courtcasematcher/model/mapper/CaseMapper.java). This would allow you to use the `Sex` enum in the domain model which would be nice but I forgot to mention this on the ticket so don't worry about it 🤦‍♂️

This works perfectly well 👍